frontend/yarn.lock

```diff
@@ -2409,7 +2433,7 @@ react-is@^18.2.0:
 
 react-password-checklist@^1.5.0:
   version "1.5.0"
-  resolved "https://registry.yarnpkg.com/react-password-checklist/-/react-password-checklist-1.5.0.tgz#b9955f90eeea1924c8e2b9f4d7ab2dc2b73ca28c"
+  resolved "https://registry.npmjs.org/react-password-checklist/-/react-password-checklist-1.5.0.tgz"
```

Comment:
**question**: Is there a reason why we changed to npmjs.org instead of yarnpkg here (since that would have been in the front-end)? Just curious for my own learning.